Source/ImPlotWidgets/Public/ImPlotWidget.h

```diff
@@ -46,9 +56,11 @@ class IMPLOTWIDGETS_API SImPlotWidget : public SCompoundWidget
 	FString Title;
 	FString XLabel;
 	FString YLabel;
+	EUnit XUnit;
+	EUnit YUnit;
 
 	float XMaxValue;
-	float YMaxValue;
+	mutable float YMaxValue;
 	float YMinValue;
 
 	mutable TArray<bool> ShowLine;
```

Comment:
I know it's not in this review but did we have a task in to remove this, I can't remember? Or removing the version in LineFilledGraphWidget.h maybe?